src/components/dialogs/network-modifications/vsc/modification/vsc-modification-dialog.tsx

```diff
@@ -223,7 +332,7 @@ const VscModificationDialog: React.FC<any> = ({
             <ModificationDialog
                 fullWidth
                 onClear={setValuesAndEmptyOthers}
-                onSave={onSubmit}
+                onSave={handleSubmit(onSubmit)}
```

Comment:
don't you have to use a callback like () => handleSubmit(onSubmit)

Reply:
handleSubmit is already a function provided by react-form-hook, it used like that in there documentation: https://react-hook-form.com/docs/useform/handlesubmit